src/legacy/core_plugins/embeddable_api/public/panel/panel_header/panel_options_menu.tsx

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Aren't these files (the UI components) the same as in the old code?
Could they be moved rather than added?
Or would it require changes?

Reply:
They are very similar but it'd be more work than it's worth, the original stuff is coupled to redux.   Plus I think it'd be weird because those panels actually use V1 style embeddables, not this V2 style.  All this should get cleaned up in subsequent PRs that switch code over to actually using this new plugin.  A downside to splitting up the giant PR this way, but I do think overall this was the better approach than that giant PR.